tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -785,6 +785,31 @@ at Samples.OpenTelemetrySdk.Program.RunSpanUpdateMethods(TelemetrySpan span),
     TraceId: Id_25,
     SpanId: Id_33,
     Name: internal,
+    Resource: SomeHttpSpan,
+    Service: MyServiceName,
+    Type: custom,
+    ParentId: Id_26,
+    Tags: {
+      env: integration_tests,
+      http.status_code: 404,
+      language: dotnet,
+      otel.library.name: MyServiceName,
+      otel.status_code: STATUS_CODE_UNSET,
```

Comment:
I assume this is an existing issue, but this seems weird

Reply:
> Hmm maybe this hasn't been correct ever for us?

Yeah, I'm sure I've seen this for a long time, it just stood out 😃

Reply:
I'll note it down to take a look some point in the future